CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.7.0]
+
+### Changed
+
+* Update `webpack` versions and dependencies
+* Update `KeepTerminal` to also include `Load app on device` and `Delete app from device`
+
+### Added
+
+* Start `Update Container` task in background
+* Add function to properly retrieve the `APPNAME`, using `make listinfo` (Fix Improve App detection [#40](https://github.com/LedgerHQ/ledger-vscode-extension/issues/40))
+
+### Fixed
+
+* Export the chosen device when opening the terminal (Fix _Export currently chosen device when opening the terminal_ [#15](https://github.com/LedgerHQ/ledger-vscode-extension/issues/15))
+* 1st CI workflows to perform spelling check and also build / Lint / Test
+  (Fix _Add "rule enforcer"_ [#6](https://github.com/LedgerHQ/ledger-vscode-extension/issues/6) and
```

Comment:
I don't think what is done in this PR is addressing the issue #6 . It's about adding a rule enforcer for apps, not for the extension itself ^^

Reply:
ok, I removed it